pkg/scheduler/framework/framework.go

```diff
@@ -40,9 +40,9 @@ const (
 
 	// The reasons to use for scheduling decisions.
 	pickedByPolicyReason                  = "picked by scheduling policy"
-	pickFixedInvalidClusterReasonTemplate = "cluster is not eligible for resource placement yet: %s"
-	pickFixedNotFoundClusterReason        = "specified cluster is not found"
-	notPickedByScoreReason                = "cluster does not score high enough"
+	pickFixedInvalidClusterReasonTemplate = "cluster \"%s\" is not eligible for resource placement yet: %s"
```

Comment:
Message is usually using upper case. Can you fix that?

eg deployment condition,

status:
  conditions:
  - lastTransitionTime: "2024-05-21T08:31:21Z"
    lastUpdateTime: "2024-05-21T08:31:24Z"
    message: ReplicaSet "member-agent-76c4999765" has successfully progressed.
    reason: NewReplicaSetAvailable
    status: "True"
    type: Progressing
  - lastTransitionTime: "2024-05-24T14:53:37Z"
    lastUpdateTime: "2024-05-24T14:53:37Z"
    message: Deployment does not have minimum availability.
    reason: MinimumReplicasUnavailable
    status: "False"
    type: Available

Reply:
Addressed on #833